Description: MNTPROC_MNT and MNTPROC_UMNT are no longer defined by the
 kernel. Hack around this by restoring the old definitions.
Upstream: http://thread.gmane.org/gmane.linux.nfs/28059 suggests a better
 fix is in progress.

diff -Nur -x '*.orig' -x '*~' klibc-1.5.15/usr/kinit/nfsmount/mount.c klibc-1.5.15.new/usr/kinit/nfsmount/mount.c
--- klibc-1.5.15/usr/kinit/nfsmount/mount.c	2009-01-04 19:28:03.000000000 +0000
+++ klibc-1.5.15.new/usr/kinit/nfsmount/mount.c	2009-10-04 22:52:44.000000000 +0100
@@ -44,6 +44,11 @@

 #define MNT_REPLY_MINSIZE (sizeof(struct rpc_reply) + sizeof(uint32_t))

+#ifndef MNTPROC_MNT
+#  define MNTPROC_MNT	1
+#  define MNTPROC_UMNT	3
+#endif /* MNTPROC_MNT */
+
 static int get_ports(uint32_t server, const struct nfs_mount_data *data)
 {
	uint32_t nfs_ver, mount_ver;
